src/Pages/TeamPage/Team.jsx

```diff
@@ -0,0 +1,338 @@
+import design from "./spam/design.svg"
+import { useState } from "react";
+const Card = (props) => {
+    return (
+        <div key={props.id} className={styles.card}>
+            <div className={styles.box}>
+                <img className={styles.img} src={props.img} alt={props.alt} />
+            </div>
+
+            <div className={styles.design}>
+                <img className={styles.desimg} src={design} />
+                <div className={styles.social}>
+                    <img className={styles.logo} src="https://s3-alpha-sig.figma.com/img/f450/f8aa/8efd2714f3a4aaccae56c77a519aa596?Expires=1705881600&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=SsqKOC8E9WXu9owo67Cu0M193CvxWusMmK~nEuxe2bH6ETo2lSbaWSkVEdg5icZLiKCP~JVGNUOnU9lQrM5LcvTFvs8-aVLsHoCYxKIL5~Kewtaqfx5SwaWMERKeGeFOO-YBGFCgEsXn-lkRPU5Ravo0CDteoA5mnY5CJfvOcofFKd5QTLCzu~qRdfTHpZFsAoz-KreK~skqcoClwJDieJ41eytTo-YJbsUTGa1HIOQOdCMmd8K7Sf71jkHf9XP1P5NkFVROK-9KWlAQDUssDl23Y5H2uWiXT4VYknyorwqYJoBpvk3-PnXoyEIbvZz6gGrR~O6EKr4jJwkTVEAzhw__" alt="instagram" />
+                    <img className={styles.logo} src="https://s3-alpha-sig.figma.com/img/1c7f/bbb4/7f72f4cd9c19126b202afe2584ed4b29?Expires=1705881600&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=i1aXekZZobz2lwcS9bGkvY0Eq1G9ciXyxeLWsijHv1IcbrT0jnyxF7m3WSWdeGFZvjerRlW63kc-0CJCe40vIxyjkAuaoGoMaY7dgbmWvI1ZB7bSvwAwZmjzji2p937PXrHDPadnjKyvSnoNoa2OlU8dRJOQBqK28srYVRl4GwtjHPr4QwmDNe9yayQSmybKvso-Y-SgaDepzLe06UfRA34Iaa--hpS-WSpvNwfy6l~dY1~~fO586pLTr5bVVOD4FQRDszss5Om8~9EQdzGMcRXbFAw3EGJ8xbv1CHAtsFuku8B0WSehTx1z2ssolcH5wWo89qBwqvdfeM3BcMO88g__" alt="facebook" />
+                    <img className={styles.logo} src="https://s3-alpha-sig.figma.com/img/ba47/8f3e/3cce233fa070836da2c8bf0bc7bdaaed?Expires=1705881600&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=K5qi0wLh~6etEM2Gpq~Cd4eJ3QOUtDi6G06ylroqQd1aDN-P1uwUOvuraRcGvGOPcbcfud2Saa5nU0K4qsLBPEoYRfDf5fZYDGCYfcUWoK0LeNMJ~VsujZ7CGcBZz5DmtTEujdLhYUcqtP3C7Ce3ZM6PvNRf7TPhFdkQXfIjJKve-2Qt26Cvjoa7PzYFxFbfbYcuHz2Jl1Tcwg3awkL5jHrBNR2ty-gVtdsP~k0bO7DArHIcGl4ri8X17xMlXZoWOStMKOI23SaGSFfEMRhQkJRUUZPWDYjbYOJBIMecempEDEgoOJlq5o55Vc9VQL40Qgau6hcvhxJ6A6~RG24Nkw__" alt="twiter" />
+                    <img className={styles.logogit} src="https://i.pinimg.com/736x/b5/1b/78/b51b78ecc9e5711274931774e433b5e6.jpg" alt="github" />
+                </div>
+            </div>
+            <div className={styles.content}>
+                <p className={styles.name}>{props.name}</p>
+                <p className={styles.designation}>{props.designation}</p>
+            </div>
+        </div>
+    );
+}
+const Teams = () => {
+    const teams = [
+        {
+            id: 1,
+            team: "FACULTY",
+        },
+        {
+            id: 2,
+            team: "ORGANISERS",
+        },
+        {
+            id: 3,
+            team: "TECH",
+        },
+        {
+            id: 4,
+            team: "CORE",
+        },
+        {
+            id: 5,
+            team: "VOLUNTEERS",
+        },
+
+    ];
```

Comment:
make separate json files for faculty, tech team, organisers, etc and use them here